client/my-sites/importer/uploading-pane.jsx

```diff
@@ -165,7 +219,36 @@ class UploadingPane extends React.PureComponent {
 					) }
 					<DropZone onFilesDrop={ isReadyForImport ? this.initiateFromDrop : noop } />
 				</div>
+				{ this.props.optionalUrl && (
+					<div className="importer__uploading-pane-url-input">
+						<FormLabel>
+							{ this.props.optionalUrl.title }
+							<TextInput
+								label={ this.props.optionalUrl.title }
+								onChange={ this.setUrl }
+								value={ this.state.urlInput }
+								placeholder="https://newsletter.substack.com"
```

Comment:
For consistency this should contain a trailing slash.
```suggestion
								placeholder="https://newsletter.substack.com/"
```

Reply:
Done, thank you! :)